doc/api/cli.md

```diff
@@ -127,6 +127,13 @@ added: v12.0.0
 
 Specify the file name of the CPU profile generated by `--cpu-prof`.
 
+### `--disallow-code-generation-from-strings`
+<!-- YAML
+added: REPLACEME
+-->
+
+Prevent strings like `eval` that generate code from functioning.
```

Comment:
We don’t typically document V8 flags, but if we do it for this one, I’d be a bit more explicit here:

```suggestion
Make built-in language features like `eval` and `new Function` that generate
code from strings throw an exception instead. This does not affect the Node.js
`vm` module.
```